### ISSUE TITLE
circleci: failed tests: TestDockerFinagle 

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#6031]
The following test appears to have failed:

[#16233](https://circleci.com/gh/cockroachdb/cockroach/16233):

```
	at org.junit.runner.JUnitCore.run(JUnitCore.java:117)
	at org.junit.runner.JUnitCore.runMain(JUnitCore.java:98)
	at org.junit.runner.JUnitCore.runMainAndExit(JUnitCore.java:53)
	at org.junit.runner.JUnitCore.main(JUnitCore.java:45)

FAILURES!!!
Tests run: 12,  Failures: 2

I160413 13:18:29.577395 acceptance/cluster/localcluster.go:649  stopping
I160413 13:18:29.577657 acceptance/cluster/localcluster.go:547  event stream done, resetting...: net/http: request canceled
I160413 13:18:29.577677 acceptance/cluster/localcluster.go:570  events monitor exits
I160413 13:18:29.578855 acceptance/cluster/docker.go:397  ContainerKill: *errors.errorString: Error response from daemon: Cannot kill container efe6ec16bb22fd4ed037eb89ba4e10166e3205a892df54971345a86b560f3610: Container efe6ec16bb22fd4ed037eb89ba4e10166e3205a892df54971345a86b560f3610 is not running
--- FAIL: TestDockerFinagle (12.77s)
	util_test.go:281: expected success: non-zero exit code: 1
=== RUN   TestGossipPeerings
I160413 13:18:31.639207 acceptance/cluster/localcluster.go:286  Initializing Cluster AdHoc 3x1:
{"name":"AdHoc 3x1","nodes":[{"count":3,"stores":[{"count":1,"max_ranges":0}]}],"duration":5000000000}
I160413 13:18:31.640859 acceptance/cluster/docker.go:102  ImagePull cockroachdb/builder:20160305-182433 already exists
I160413 13:18:32.034886 acceptance/cluster/localcluster.go:581  creating certs (1024bit) in: /home/ubuntu/cockroach/.localcluster.certs.591413409
I160413 13:18:32.115770 acceptance/cluster/localcluster.go:532  events monitor starts
I160413 13:18:32.385533 acceptance/cluster/localcluster.go:480  *** started roach0 ***
  ui:        https://127.0.0.1:32828
  trace:     https://127.0.0.1:32828/debug/requests
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:30: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	util_test.go:65: running in docker mode
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

**Comment by tschottdorf** [comment]
```
1) A postgres client should select rows via a prepared query(com.twitter.finagle.postgres.integration.IntegrationSpec)
org.scalatest.exceptions.TestFailedException: 0 did not equal 2
```

**Comment by dt** [comment]
dupe of #5951, which I'm still hoping will be resolved when #5484 is fixed
